### DISCUSSION TITLE
Capacitor 1.x Documentation

### DISCUSSION THREAD

**Discussion by tonyawad88** [ionic-team/capacitor#3514] [Q&A] [answered]
Hello,

Now that the capacitor website has moved to 2.x docs. 

Can you please share the link for the 1.x docs? or maybe if we can have the 2.3.0 as a dropdown so we can pick the capacitor version we're in and get the proper docs? 

The reason why I am asking, we are running cap v1.3.x and for example App.removeListener doesn't seem to exist.

Thank you!

**Comment by jcesarmobile** [comment, marked as answer]
As far as I know it's a planned feature on https://github.com/ionic-team/capacitor-site, but it's not possible at the moment, will be probably available for capacitor 3 release.

But removeListener has never worked and was removed from the docs long ago, see https://github.com/ionic-team/capacitor/issues/1534#issuecomment-494371744 for more details

**Reply by tonyawad88** [reply]
Thank you!
